src/Microsoft.ML.Api/GenerateCodeCommand.cs

```diff
@@ -139,7 +139,6 @@ public void Run()
                     new Dictionary<string, string>
                     {
                         { "EXAMPLE_CLASS_DECL", loaderSb.ToString() },
-                        { "SCORED_EXAMPLE_CLASS_DECL", nonScoreSb.ToString() },
```

Comment:
>SCORED_EXAMPLE_CLASS_DECL [](start = 27, length = 25)

In our internal repo we actually have this thing

          // These are all remaining available columns, either supplied as the input, or intermediate
            // columns generated by the transforms. Materializing these columns has a performance cost,
            // so they are commented out. Feel free to uncomment any column that is useful for your scenario.
#if false
/*#SCORED_EXAMPLE_CLASS_DECL#*/
/*#/SCORED_EXAMPLE_CLASS_DECL#*/
#endif

Are you sure you want to delete it?

Reply:
Yep, that's strange.